content/docs/state-and-lifecycle.md

```diff
@@ -68,23 +68,23 @@ ReactDOM.render(
 );
 ```
 
-To implement this, we need to add "state" to the `Clock` component.
+यसलाई कार्यान्वयन गर्न हामीले `Clock`कम्पोनेंटमा "state" थप्न आवश्यक छ।
 
-State is similar to props, but it is private and fully controlled by the component.
+State Props जस्तै छ तर यो निजी हुन्छ​ र पूरै कम्पोनेंटद्वारा नियन्त्रित हुन्छ​।
 
-## Converting a Function to a Class {#converting-a-function-to-a-class}
+## एउटा functionलाई classमा रूपान्तरण गर्ने तरिका{#converting-a-function-to-a-class}
 
-You can convert a function component like `Clock` to a class in five steps:
+तपाईं `Clock` जस्तो function कम्पोनेंटलाई classमा पाँच चरणमा रूपान्तरण गर्न सक्नुहुन्छ।
 
-1. Create an [ES6 class](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes), with the same name, that extends `React.Component`.
+१. [ES6 class](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes) सिर्जना गर्नुहोस् उही नामको साथ जसले `React.Component` विस्तार गर्दछ।
 
-2. Add a single empty method to it called `render()`.
+२. यसमा `render()` नाम भएको एक खाली खाली मेथड थप्नुहोस्।
 
-3. Move the body of the function into the `render()` method.
+३. `render()` मेथडमा functionको body सार्नुहोस्।
 
-4. Replace `props` with `this.props` in the `render()` body.
+४. `render()`को bodyमा `props`लाई `this.props`ले बदल्नुहोस्.
```

Comment:
How about `render() method भित्र` instead of  `render()को bodyमा` -- I think translating the intent is more important than translating word to word